manim/mobject/geometry/line.py

```diff
@@ -1015,6 +1015,38 @@ def construct(self):
             return self.angle_value / DEGREES
         return self.angle_value
 
+    @staticmethod
+    def from_three_points(
+        A: np.ndarray, B: np.ndarray, C: np.ndarray, **kwargs
+    ) -> Angle:
+        """The angle between the lines AB and BC.
+
+        This constructs the angle :math:`\angle ABC`.
+
+        Parameters
+        ----------
+        A: The endpoint of the first angle leg
+        B: The vertex of the angle
+        C: The endpoint of the second angle leg
```

Comment:
```suggestion
        A
            The endpoint of the first angle leg
        B
            The vertex of the angle
        C
            The endpoint of the second angle leg
```

The formatting in my comment was not meant to be used verbatim in ReST

Reply:
Oh, I see, I should have looked at the other instances of it. My bad!